src/wallet/rpcwallet.cpp

```diff
@@ -2128,16 +2129,15 @@ UniValue lockunspent(const UniValue& params, bool fHelp)
             + HelpExampleCli("listlockunspent", "") +
             "\nUnlock the transaction again\n"
             + HelpExampleCli("lockunspent", "true \"[{\\\"txid\\\":\\\"a08e6907dbbd3d809776dbfc5d82e371b764ed838b5655e72f463568df1aadf0\\\",\\\"vout\\\":1}]\"") +
+            "\nUnlock all current locked transaction outputs\n"
+            + HelpExampleCli("lockunspent", "true") +
             "\nAs a json rpc call\n"
             + HelpExampleRpc("lockunspent", "false, \"[{\\\"txid\\\":\\\"a08e6907dbbd3d809776dbfc5d82e371b764ed838b5655e72f463568df1aadf0\\\",\\\"vout\\\":1}]\"")
         );
 
     LOCK2(cs_main, pwalletMain->cs_wallet);
 
-    if (params.size() == 1)
-        RPCTypeCheck(params, boost::assign::list_of(UniValue::VBOOL));
-    else
-        RPCTypeCheck(params, boost::assign::list_of(UniValue::VBOOL)(UniValue::VARR));
+    RPCTypeCheck(params, boost::assign::list_of(UniValue::VBOOL)(UniValue::VARR), true);
```

Comment:
This is not correct, I think.
Setting `fAllowNull` to true is different to allowing one or two arguments.